paywall/src/unlock.js/web3Proxy.ts

```diff
@@ -74,22 +74,27 @@ export function handleWeb3Call({
     case 'eth_accounts':
       postMessage('data', PostMessages.WEB3_RESULT, {
         id,
-        error: null,
-        result: { id, jsonrpc: '2.0', result: [proxyAccount] },
+        jsonrpc: '2.0',
+        result: {
+          id,
+          jsonrpc: '2.0',
+          // if account is null, we have no account, so return []
+          result: proxyAccount ? [proxyAccount] : [],
```

Comment:
This is the piece that fixes
> return [] instead of [null] when there is no account.
?

Just want to make sure I see what's happening where

Reply:
yes that lives here.